FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -167,17 +220,43 @@ main()
 
 	get_options "$@"
 
-	local directory=${XNAT_PBS_JOBS_RUNNING_STATUS_DIR}/${g_project}
-	local file="FunctionalPreprocessing.${g_subject}_${g_classifier}_${g_scan}.RUNNING"
+	local directory="${XNAT_PBS_JOBS_BUILD_DIR}/${g_project}/FunctionalPreprocessing.${g_session}_${g_scan}_RUNNING_STATUS"
+	local file="FunctionalPreprocessing.${g_session}_${g_scan}.RUNNING"
+	
+	local existing_file="${XNAT_PBS_JOBS_ARCHIVE_ROOT}/${g_project}/arc001/${g_session}/RESOURCES/RunningStatus/${file}"	
 	local path=${directory}/${file}
-
+	local g_dir=${directory}
 	if [ "${g_running}" = "TRUE" ]; then
 		mkdir -p ${directory}
 		touch ${path}
+		${XNAT_PBS_JOBS}/WorkingDirPut/PutDirIntoResource.sh \
+					--user=${g_user} \
+					--password=${g_password} \
+					--server=${g_server} \
+					--project=${g_project} \
+					--subject=${g_subject} \
+					--session=${g_session} \
+					--resource=${g_resource} \
+					--reason=${g_reason} \
+					--dir=${g_dir} \
+					--use-http \
+					--force	
+			rm -rf ${directory}
 
 	else
-		if [ -e "${path}" ]; then
-			rm -f ${path}
+		if [ -e "${existing_file}" ]; then
+			echo ${g_resource}
```

Comment:
I assume that you initially echoed these values `g_resource` and `file` to help you with debugging. But they aren't needed for anything now are they? If you'd like to leave them in for future debugging purposes, I suggest you turn them into calls to the `log_Debug_Msg()' function. Like the `log_Msg` function, the `log_Debug_Msg` function is defined in the `log.shlib` file. It will only output the specified message if debugging mode is on.

Reply:
deleted